docs/contributing/python_guidelines.rst

```diff
@@ -16,6 +16,8 @@ In addition, import lines should be sorted according to `isort <https://pycqa.gi
 If you have installed Wagtail's testing dependencies (``pip install -e '.[testing]'``), you can check your code by
 running ``make lint``.
 
+You can run all Python formatting manually via ``make format``.
```

Comment:
'manually' feels like an odd word to use here... I understand it's to distinguish it from the pre-commit hook, but this command is still very much automating a manual task. I'd be inclined to just leave it out:

```suggestion
You can run all Python formatting with ``make format``.
```